### ISSUE TITLE
SLIME Typeout Frame Doesn't Receive slime-autodoc messages.

### ISSUE THREAD

**Issue by chebert** [slime/slime#221]
It could be that I'm not setting up the typeout frame correctly or it isn't supported.

I really just want it to hold the argument lists for functions (because it's really finicky and shows for less than a second in the echo buffer) from slime-autodoc. 

So far the only way I can get the typeout frame to show anything at all is to evaluate `(slime-typeout-message "Message")` which causes the text "Message" to appear.

Emacs Version: 24.4.1 

SBCL Version: 1.2.6

`(slime-changelog-date)`: 2014-12-23

```
(progn
  ;; (ql:quickload :quicklisp-slime-helper)
  (load (expand-file-name "~/quicklisp/slime-helper.el"))
  ;; Replace "sbcl" with the path to your implementation
  (setq inferior-lisp-program "sbcl")
  (setq slime-contribs '(slime-fancy slime-typeout-frame))

  (add-hook 'slime-mode-hook (lambda ()
                   (require 'slime-typeout-frame)
                   (slime-ensure-typeout-frame)
                   ))

  ;; Start slime when editing lisp.
  (add-hook 'lisp-mode-hook (lambda () (slime-mode t)))
  (add-hook 'inferior-lisp-mode-hook (lambda () (inferior-slime-mode t))))
```

**Comment by chebert** [comment]
If someone knows if this is not an issue with SLIME, could you please let me know where I should post this? I'm also happy to try to debug this myself, but it'd be great if someone could point me in the right direction, as I have very little experience with Emacs Lisp.

**Comment by capitaomorte** [comment]
@chebert. Perhaps you don't need the typeout frame

```
(add-hook 'slime-autodoc-mode-hook
          (lambda ()
            (setq eldoc-documentation-function
                  (lambda (&rest args)
                    (let ((docstring (apply #'slime-autodoc args)))
                      (when docstring
                        (with-current-buffer (get-buffer-create "*typeout*")
                          (goto-char (point-max))
                          (insert docstring "\n"))))))))
```

I briefly tested this with SLY but something similar should work with SLIME.

**Comment by chebert** [comment]
Just tried it and it works! 

Thank you, I've been wanting this to work for a while now :)

**Comment by luismbo** [comment]
@chebert I think I've fixed this issue in 00b09ed. Let me know if it works for you. Thanks.
